File: iOS/Objects/src/B4iProject/LS_frmlogin.m

#import "iCore.h"
@interface LS_frmlogin:NSObject
@end

@implementation LS_frmlogin

- (void)LS_general:(B4ILayoutData*)views :(int)width :(int)height{
[B4ILayoutBuilder setScaleRate:0.3];
[B4ILayoutBuilder scaleAll:views :width :height];
[B4ILayoutBuilder setScaleRate:0.0];
[views get:@"pnlloading1"].Left = (int)((50 / 100.0 * width) - ([views get:@"pnlloading1"].Width / 2));
[views get:@"lblforgot"].Width = (int)(([views get:@"btnlogin"].Width));
[views get:@"lblforgot"].Left = (int)((50 / 100.0 * width) - ([views get:@"lblforgot"].Width / 2));
//BA.debugLineNum = 8;BA.debugLine="btnguest.HorizontalCenter = 50%x"[frmlogin/General script]
[views get:@"btnguest"].Left = (int)((50 / 100.0 * width) - ([views get:@"btnguest"].Width / 2));
//BA.debugLineNum = 9;BA.debugLine="btnlogin.HorizontalCenter = 50%x"[frmlogin/General script]
[views get:@"btnlogin"].Left = (int)((50 / 100.0 * width) - ([views get:@"btnlogin"].Width / 2));
//BA.debugLineNum = 10;BA.debugLine="pnlloading1.VerticalCenter = 38%y"[frmlogin/General script]
[views get:@"pnlloading1"].Top = (int)((38 / 100.0 * height) - ([views get:@"pnlloading1"].Height / 2));

}
@end

File: Attachment/Manage App/Android/Objects/src/com/apps/manageropencart/designerscripts/LS_frmlogin.java
package com.apps.manageropencart.designerscripts;
import anywheresoftware.b4a.objects.TextViewWrapper;
import anywheresoftware.b4a.objects.ImageViewWrapper;
import anywheresoftware.b4a.BA;


public class LS_frmlogin{

public static void LS_general(java.util.LinkedHashMap<String, anywheresoftware.b4a.keywords.LayoutBuilder.ViewWrapperAndAnchor> views, int width, int height, float scale) {
anywheresoftware.b4a.keywords.LayoutBuilder.setScaleRate(0.3);
anywheresoftware.b4a.keywords.LayoutBuilder.scaleAll(views);
views.get("pnlfield").vw.setLeft((int)((50d / 100 * width) - (views.get("pnlfield").vw.getWidth() / 2)));
//BA.debugLineNum = 4;BA.debugLine="pnlfield.VerticalCenter = 65%y"[frmlogin/General script]
views.get("pnlfield").vw.setTop((int)((65d / 100 * height) - (views.get("pnlfield").vw.getHeight() / 2)));
//BA.debugLineNum = 5;BA.debugLine="lbllog.HorizontalCenter = 50%x"[frmlogin/General script]
views.get("lbllog").vw.setLeft((int)((50d / 100 * width) - (views.get("lbllog").vw.getWidth() / 2)));
//BA.debugLineNum = 6;BA.debugLine="lbllog.VerticalCenter = 50%y"[frmlogin/General script]
views.get("lbllog").vw.setTop((int)((50d / 100 * height) - (views.get("lbllog").vw.getHeight() / 2)));

}
}